src/main/java/org/isf/medicalstock/gui/MovStockMultipleCharging.java

```diff
@@ -998,7 +1000,8 @@ public Component getTableCellRendererComponent(JTable table, Object value, boole
 		public Component getTableCellRendererComponent(JTable table, Object value, boolean isSelected, boolean hasFocus, int row, int column) {
 			// First format the cell value as required
 			Component cell = super.getTableCellRendererComponent(table, value, isSelected, hasFocus, row, column);
-			value = formatter.format((Number) value);
+			if (value != null)
+				value = formatter.format((Number) value);
```

Comment:
The cast to `Number` is not really needed and is redundant.  (SonarLint)

Reply:
Removed